project/Assignment.cs

```diff
@@ -2,19 +2,76 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using assignment_3;
 
 namespace assignment_3
 {
     public interface IAssignment
     {
         public int AssignmentID { get; }
-        public static int nextId;
+        public static int nextId; 
         public string Topic { get; set; }
-
         public DateTime DueDate { get; set; }
         public DateTime? SubmissionDate { get; set; }
+        public Student SubmittingStudent { get; set; }
+    }
 
+    public class Assignment : IAssignment
```

Comment:
Why are you creating a concrete class called Assignment if it's supposed to be represented as an interface? The inheritors are IndividualProject, GroupProject, Essay and Coding.

Reply:
sorry my bad . i thought its different . u are right